.github/workflows/playwright.yml

```
@@ -385,13 +401,63 @@ jobs:
           echo "Install Playwright Browsers"
           npx playwright install --with-deps
 
-      - name: Run Playwright tests
+      - name: Copy changes from PR
+        run: |
+          echo "Commit Message: ${{ env.COMMIT_MSG }}"
+          echo "Setting up some git config..."
+          git config --global user.email "cms-platform+sage-testing@pantheon.io"
+          git config --global user.name "Pantheon WPCM Bot"
+
+          echo "Switching to git mode..."
+          terminus connection:set "${{ env.SITE_ID }}".dev git
+
+          echo "Clone the site locally and copy PR updates"
+          terminus local:clone ${{ env.SITE_ID }}
+          cd ~/pantheon-local-copies/"${{ env.SITE_ID }}"
+
+          echo "Copying latest changes and committing to the site."
+          rsync -a --exclude='.git' --exclude='status-*.txt' --exclude="node_modules" "${{ github.workspace }}/" .
+          git add -A
+          git commit -m "Update to latest commit: ${{ env.COMMIT_MSG }}" || true
+
+          echo "Installing wp-graphql..."
+          composer require wp-graphql/wp-graphql
+          git add composer.json composer.lock
+          git commit -m "Add WP-GraphQL plugin" || true
+
+          echo "Copying the subdomain multisite config/application.php file..."
+          cp -f "${{ github.workspace }}/.github/fixtures/config/application.subdom.php" config/application.php
+          git add config/application.php
+          git commit -m "Add subdomain multisite config" || true
+
+          # Push and wait for the git push to finish.
+          git push origin master || true
+          terminus workflow:wait "${{ env.SITE_ID }}".dev
+
+          echo "Checking WordPress install status"
+          terminus wp "${{ env.SITE_ID }}".dev -- cli info
+
+          # Activte WP-GraphQL plugin
+          terminus wp "${{ env.SITE_ID }}".dev -- plugin activate wp-graphql
+          terminus env:clear-cache "${{ env.SITE_ID }}".dev
+
+          # Run curl checks against the site URLs to ensure the main site and the subdomain site exist.
+          echo "Checking site URLs"
+          SITE_URL_TEST=$(curl -s -o /dev/null -w "%{http_code}" ${{ env.SITE_URL }})
+          SUBDOMAIN_URL_TEST=$(curl -s -o /dev/null -w "%{http_code}" ${{ env.SUBDOMAIN_URL }})
+          if [ $SITE_URL_TEST -ne 200 ] || [ $SUBDOMAIN_URL_TEST -ne 200 ]; then
+            echo "One or more site URLs are not returning a 200 status code. Exiting."
+            exit 1
+          fi
```

might be useful to debug print what each are
```suggestion
          if [ $SITE_URL_TEST -ne 200 ] || [ $SUBDOMAIN_URL_TEST -ne 200 ]; then
            echo "One or more site URLs are not returning a 200 status code. Exiting."
            echo "${{ env.SITE_URL }} - ${SITE_URL_TEST}"
            echo "${{ env.SUBDOMAIN_URL }} - ${SUBDOMAIN_URL_TEST}"
            exit 1
          fi
```